src/components/data-exploration/differential-expression-tool/AdvancedFilteringModal.jsx

```diff
@@ -1,53 +1,55 @@
 import React from 'react';
 import {
-  Modal, Form, Button, Space, Select, InputNumber, Dropdown, Menu, Tooltip,
+  Modal, Form, Button, Space, Select, InputNumber, Dropdown, Menu,
 } from 'antd';
 import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
 import { CloseOutlined, PlusOutlined } from '@ant-design/icons';
 
 const AdvancedFilteringModal = (props) => {
-  const { onCancel } = props;
+  const { onCancel, onLaunch } = props;
   const [form] = Form.useForm();
+  const advancedFilters = useSelector((state) => state.differentialExpression.comparison.advancedFilters) || [];
 
-  const criteriaOptions = [
-    { value: 'logfc', label: 'logFC' },
-    { value: 'pValue', label: 'adj p-value' },
-    { value: 'pct1', label: 'Pct1' },
-    { value: 'pct2', label: 'Pct2' },
+  const columnNameOptions = [
```

Comment:
I would keep criteria, or perhaps `propertyOptions` / `attributeOptions`?

Reply:
By the way all of these constants (`columnNameOptions`, `comparisonOptions `, `valueRestrictions`, `presetFilters `) are never going to change, you could leave them outside the `AdvancedFilteringModal`

Reply:
Why would we want them outside? There are not used anywhere, will make the codebase more littered with more files.

Reply:
Plus its easier to see them in the same file not to follow them across different ones

Reply:
not outside in another file, I mean outside the function.